src/components/Heading.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Please have a look at this code quality issue. As the import happens automatically, you do not need this line here :) 
```suggestion

```